src/components/userProfile/components/UserInfo.js

```diff
@@ -66,3 +128,18 @@ export default class UserInfo extends React.Component {
         );
     }
 }
+
+UserInfo.propTypes = {
+    actions: PropTypes.object,
+    userData: PropTypes.object,
+};
+
+const mapStateToProps = state => ({
+    userData: getUserData(state),
+});
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
remove the function `mapDispatchToProps`

Reply:
ok